environments/development.env

```diff
@@ -8,3 +8,7 @@
 
 NODE_ENV=development
 CONFIG_NAME=development
+
+#
+# We need to keep storage path
+STORAGE_PATH=/temp/TheCoin/tests/dbs/
```

Comment:
this storage path is a real pain.  It's been mostly refactored away to be unnecessary, but the remaining reasons are very hard to eliminate